src/standalone/Soil/energy_hydrology.jl

```diff
@@ -62,10 +62,16 @@ Base.@kwdef struct EnergyHydrologyParameters{
     γ::FT
     "Reference temperature for the viscosity factor"
     γT_ref::FT
-    "Soil PAR Albedo"
-    PAR_albedo::SF
-    "Soil NIR Albedo"
-    NIR_albedo::SF
+    "Soil PAR Albedo dry"
+    PAR_albedo_dry::SF
+    "Soil NIR Albedo dry"
+    NIR_albedo_dry::SF
+    "Soil PAR Albedo wet"
+    PAR_albedo_wet::SF
+    "Soil NIR Albedo wet"
+    NIR_albedo_wet::SF
+    "Thickness of top of soil used in albedo calculations (m)"
+    top_depth::FT
```

Comment:
Do we want to package these into a struct? Maybe just the albedos? The only advantage I can see to that is possibly a cleaner way of supporting a mix of floats and fields.

Reply:
I am opening an issue for this, we dont need to do it here.